deploy/task-cad-checks.yaml

```diff
@@ -32,14 +32,14 @@ spec:
       valueFrom:
         secretKeyRef:
           name: cad-pd-token
-          key: token
+          key: CAD_PD_TOKEN
```

Comment:
this will load the ENVVAR, but it will be used as CAD_PD (see line 34)
you can use the `envFrom` format to load the whole secret as envvars